picoCTF-web/api/apps/v1/teams.py

```diff
@@ -40,6 +40,9 @@ def post(self):
                 "()+-,#'&!?", status_code=400
             )
 
+        if req['team_name'] == curr_user['username']:
```

Comment:
This seems redundant since `create_and_join_new_team()` will raise a 409 in this case

Reply:
Similar to the other exception raised, I think it's more along lines of a more clear error message than "team is already taken" since one wouldn't expect a team with one's username exists.

Reply:
Fair point; lgtm then